utils/vtn.py

```diff
@@ -0,0 +1,87 @@
+import asyncio
```

Comment:
Add documentation to what events are happening with this ven.

Will this vtn be standard or our ipkeys equivilent.  How is security handled? Since http I assume its not.

Reply:
This VTN is a simple toy VTN deployed locally on the host. It is provided to assist users in testing out the OpenADRVen agent without having to setup or pay for a VTN on a remote host. 

This VTN is hosted localhost using HTTP, which is the [default behavior of this VTN](https://openleadr.org/docs/api/openleadr.html#module-openleadr.server). Since the purpose of this toy VTN is to quickly help users test out the OpenADRVen Agent, security is not handled (i.e TLS is not configured). 

This VTN will eventually be used in testing so that we can run tests automatically either on a local environment or as part of CI. Also, IPKeys is a pain to automate especially for testing.